analytical/templatetags/piwik.py

```diff
@@ -14,6 +14,8 @@
 from analytical.utils import (is_internal_ip, disable_html,
                               get_required_setting, get_identity)
 
+import warnings
+warnings.warn('The Piwik module will be deprecated', DeprecationWarning)
```

Comment:
Maybe direct users to the Matomo module? (Also, nitpick, I think the Piwik module _is_ deprecated, not _will be_ deprecated.)

Reply:
Clarified, thanks.